en/docs/use-cases/streaming-usecase/handling-errors.md

```diff
@@ -123,7 +123,7 @@ To try out storing errors in the store, follow the steps below:
 
 2. Download the [productions.csv file](https://github.com/wso2/docs-ei/blob/master/en/streaming-integrator/docs/examples/resources/productions.csv) and save it in a location of your choice in your machine.
 
-3. [Create a Siddhi application](../develop/streaming-apps/creating-a-Siddhi-Application.md) as follows and [deploy it in the Streaming Integrator server](../develop/streaming-apps/deploying-Streaming-Applications.md).
+3. [Create a Siddhi application](../../develop/streaming-apps/creating-a-Siddhi-Application.md) as follows and [deploy it in the Streaming Integrator server](../../develop/streaming-apps/deploying-Streaming-Applications.md).
```

Comment:
@RukshiW use {{base_path}} in the image paths and avoid having capital letters in the file name.